AMW_business/src/main/java/ch/puzzle/itc/mobiliar/business/deploy/entity/DeploymentEntity.java

```diff
@@ -250,6 +250,31 @@ public String toString() {
     private List<DeploymentParameter> deploymentParameters;
 
     @Getter
+    @Setter
+    @Column(name = "ex_context_id")
+    private Integer exContextId;
+
+    @Getter
+    @Setter
+    @Column(name = "ex_release_id")
+    private Integer exReleaseId;
+
+    @Getter
+    @Setter
+    @Column(name = "ex_resource_id")
+    private Integer exResourceId;
+
+    @Getter
+    @Setter
+    @Column(name = "ex_resourcegroup_id")
+    private Integer exResourcegroupId;
+
+    @Getter
+    @Setter
+    @Column(name = "ex_runtime_resource_id")
+    private Integer exRuntimeResourceId;
```

Comment:
@yvespp Yes, without them as reference it would not be possible to display the original properties of the deleted entities. (Like the name of an environment or of an app server that was deleted)